label_studio/tasks/models.py

```diff
@@ -189,13 +189,15 @@ def get_locked_by(cls, user, project=None, tasks=None):
         """Retrieve the task locked by specified user. Returns None if the specified user didn't lock anything."""
         lock = None
         if project is not None:
-            lock = TaskLock.objects.filter(
-                user=user, expire_at__gt=now(), task__project=project
-            ).first()
+            lock = fast_first(
+                TaskLock.objects.filter(
+                    user=user, expire_at__gt=now(), task__project=project
+                )
+            )
         elif tasks is not None:
-            locked_task = tasks.filter(
-                locks__user=user, locks__expire_at__gt=now()
-            ).first()
```

Comment:
Note: only this one used `[:1]` prior to #4588. The others don't appear to depend on the primary key ordering `first()` would use.